source/hot-backup.rst

```diff
@@ -31,6 +35,8 @@ To take a hot backup of the database in your current ``dbpath``, do the followin
      > db.runCommand({createBackup: 1, backupDir: "<backup_data_path>"})
      { "ok" : 1 }
 
+The backup start time is the time the ``createBackup`` command starts running.
```

Comment:
The "start time" does not explain what data will be in the backup. Consider following rewording:
The backup is essentially a snapshot of the instance's data at the moment of createBackup command start.